analysis.ipynb

```diff
@@ -0,0 +1,1563 @@
+{
+  "nbformat": 4,
```

Comment:
The dataset should be downloaded directly from the website with pandas instead of loading locally.